packages/core/contracts/common/implementation/Testable.sol

```diff
@@ -46,7 +46,7 @@ abstract contract Testable {
         if (timerAddress != address(0x0)) {
             return Timer(timerAddress).getCurrentTime();
         } else {
-            return now; // solhint-disable-line not-rely-on-time
+            return block.timestamp; // solhint-disable-line not-rely-on-time
```

Comment:
curious what the difference is between block.timestamp and now, and also does it have any effects on tests

Reply:
they are the same thing I believe. after version 7 the `block.timestamp` syntax is preferred.